docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+- [[Status] Needs More Info](https://github.com/WordPress/gutenberg/labels/%5BStatus%5D%20Needs%20More%20Info) - The issue needs more information in order to be actionable and relevant. Typically this requires follow-up from the original reporter.
+- [Needs Product Decision](https://github.com/WordPress/gutenberg/labels/Needs%20Product%20Decision) - A person with decision-making authority needs to decide whether the project should support this issue.
+
+[Check out the label directory](https://github.com/WordPress/gutenberg/labels) for a listing of all labels.
+
+**Milestones**
+
+We put issues into [milestones](https://github.com/wordpress/gutenberg/milestones) to better categorize them. Here are some you might see:
+
+- The next 2 releases we have milestones for (e.g. 2.2, 2.3).
+- [Feature Complete](https://github.com/WordPress/gutenberg/milestone/8): This includes big features and is what will be managing the vision of Gutenberg. All of this would be done before even merge proposal is thought about. Examples here include nesting, drag and drop and extensibility API.
+- [Merge Proposal](https://github.com/WordPress/gutenberg/milestone/22): This will grow over time and include for now most likely bugs and API issues.
+- [Bonus Features](https://github.com/WordPress/gutenberg/milestone/32): Again likely not part of triage and includes nice to haves for the project, if time before merge. A few examples include collaborative editing and footnotes.
+- [Future](https://github.com/WordPress/gutenberg/milestone/35): this is something that is confirmed by everyone as a good thing but doesn’t fall into other criteria.
+
+**Triaging Issues**
+
+To keep the issue backlog healthy, it needs to be triaged regularly. *Triage* is the practice of reviewing existing issues to make sure they’re relevant, actionable, and have all the information they need.
+
+Anyone can help triage the backlog, although you’ll need contributor permission on the Gutenberg repository to modify an issue’s labels or edit its title.
+
+Here are a couple places you can start:
+
+- [All Gutenberg issues without an assigned label](https://github.com/wordpress/gutenberg/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc+no%3Alabel)
+- [The least recently updated Gutenberg issues](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc)
+
+When reviewing the issue backlog, here are some steps you can perform:
+
+- If it’s a bug report, test to confirm the report.
+- If the issue is missing labels, add some to better categorize it.
+- If the issue is duplicate of another already in the backlog, close the issue by commenting with “Duplicate of #<original-id>”. Add any relevant new details to the existing issue.
+- If there was conversation on the issue but no actionable steps identified, follow up with the participants to see what’s actionable.
```

Comment:
There's now also a "Stale" label, to help identify issues which might need to be closed. Wondering if this is redundant with "Needs More Info" though; the latter might not be as great a fit for pull requests.

cc @youknowriad (I believe you added the label?)

Reply:
It maybe is a matter of context but 'needs more info' to me means literally that. For example, what browser or device someone was using.

Reply:
I added the `Stale` label to simplify triaging. It's difficult to find out which PRs were already reviewed otherwise. I think about adding another label to mark that PR was triaged and needs a review to proceed. I'm not fully sold on the latter though.